website/docs/tutorials/08-cairo-1-support/01-requirements.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_label: Requirements
+---
+
+# Requirements
+
+## Project structure
+
+:::note
+Cairo 1 is constantly being updated, so all the requirements mentioned here may change in the future.
+:::
+
+Currently, Cairo1 forces the following file structure of a contract:
+```
+my-contract/
+├── lib.cairo
+├── cairo_project.toml
+```
```

Comment:
It may be worth mentioning that `-` in module names are not allowed even in plain cairo compiler afaik.

https://docs.swmansion.com/scarb/docs/reference/manifest#name

Reply:
[address feedback](https://github.com/software-mansion/protostar/pull/1753/commits/23348270e515aed2e74461a421b7fcc240fb1993)